src/components/tx/TxHistory/List/TxView.js

```diff
@@ -83,4 +105,8 @@ TxView.propTypes = {
   fromAccount: PropTypes.object.isRequired,
   openTx: PropTypes.func.isRequired,
   refreshTx: PropTypes.func.isRequired,
+  currentBlock: PropTypes.string.isRequired,
```

Comment:
Didn't find usage of currentBlock properties, instead we have `network` property not declared here